{
	"id": "CVE-2024-4029",
	"vulnerabilities": [
		{
			"content": {
				"id": "CVE-2024-4029",
				"description": "A vulnerability was found in Wildflyâ€™s management interface. Due to the lack of limitation of sockets for the management interface, it may be possible to cause a denial of service hitting the nofile limit as there is no possibility to configure or set a maximum number of connections.",
				"severity": [
					{
						"type": "cvss_v31",
						"source": "secalert@redhat.com",
						"cvss_v31": {
							"vector": "CVSS:3.1/AV:L/AC:H/PR:H/UI:N/S:U/C:N/I:N/A:H",
							"base_score": 4.1,
							"base_severity": "MEDIUM",
							"temporal_score": 4.1,
							"temporal_severity": "MEDIUM",
							"environmental_score": 4.1,
							"environmental_severity": "MEDIUM"
						}
					}
				],
				"cwe": [
					{
						"source": "secalert@redhat.com",
						"cwe": [
							"CWE-770"
						]
					}
				],
				"references": [
					{
						"source": "nvd.nist.gov",
						"url": "https://nvd.nist.gov/vuln/detail/CVE-2024-4029"
					},
					{
						"source": "secalert@redhat.com",
						"url": "https://access.redhat.com/errata/RHSA-2024:8075"
					},
					{
						"source": "secalert@redhat.com",
						"url": "https://access.redhat.com/errata/RHSA-2024:8076"
					},
					{
						"source": "secalert@redhat.com",
						"url": "https://access.redhat.com/errata/RHSA-2024:8077"
					},
					{
						"source": "secalert@redhat.com",
						"url": "https://access.redhat.com/errata/RHSA-2024:8080"
					},
					{
						"source": "secalert@redhat.com",
						"url": "https://access.redhat.com/errata/RHSA-2024:8823"
					},
					{
						"source": "secalert@redhat.com",
						"url": "https://access.redhat.com/errata/RHSA-2024:8824"
					},
					{
						"source": "secalert@redhat.com",
						"url": "https://access.redhat.com/errata/RHSA-2024:8826"
					},
					{
						"source": "secalert@redhat.com",
						"url": "https://access.redhat.com/security/cve/CVE-2024-4029"
					},
					{
						"source": "secalert@redhat.com",
						"url": "https://bugzilla.redhat.com/show_bug.cgi?id=2278615"
					}
				],
				"published": "2024-05-02T15:15:07.227Z",
				"modified": "2024-11-05T02:15:04.543Z"
			},
			"ecosystems": [
				"cpe"
			]
		}
	],
	"data_source": {
		"id": "nvd-api-cve",
		"raws": [
			"vuls-data-raw-nvd-api-cve/2024/CVE-2024-4029.json"
		]
	}
}
